### ISSUE TITLE
Tags not working?

### ISSUE THREAD

**Issue by cschotke** [logstash-plugins/logstash-output-pagerduty#3]
I have been using this output for a while and noticed Tags is always empty in my PagerDuty events. I'm assuming this is supposed to list all tags for the logstash event? If so, should the config use 

```
pd_event[:details][:tags] = event["tags"] if event["tags"]
```

instead of 

```
pd_event[:details][:tags] = @tags if @tags
```

?

**Comment by suyograo** [comment]
@cschotke `tags` set upstream should be propagated to PD. You don't have to explicitly set it

**Comment by cschotke** [comment]
It never worked for me until I changed that line in the code. Has anyone else had this issue?

**Comment by suyograo** [comment]
yep, this is a bug.. the `@tags` there is referring to https://github.com/elastic/logstash/blob/master/lib/logstash/outputs/base.rb#L23

which is completely different

**Comment by suyograo** [comment]
@cschotke would you mind creating a PR to add your line? and some tests if you can mock the output? Let me know if you need help with tests

**Comment by cschotke** [comment]
@suyograo Sure, I can submit a PR. Can you provide guidance on the tests? Thanks.
